tests/Makefile.tests

```diff
@@ -13,30 +13,25 @@ CFLAGS+=-I$(SOLO5_DIR)
 %.o: %.c $(HEADERS)
 	$(CC) $(CFLAGS) -c $< -o $@
 
-ifdef UKVM_TARGETS
 Makefile.ukvm: $(UKVM_SRC)/ukvm-configure
 	$(UKVM_SRC)/ukvm-configure $(UKVM_SRC) $(UKVM_MODULES)
 
-include Makefile.ukvm
+.PHONY: ukvm-clean
+-include Makefile.ukvm
```

Comment:
`-` at front always means "don't bail if unsuccessful", which means for me that it tries (and fails) to build `Makefile.ukvm` for inclusion, but continues with the rest of the process... I'm not a gmake expert, and strongly believe there are better solutions :)